<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>PUBLICATIONS</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 52px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 57px;
  margin-top: -57px;
}
.section h2 {
  padding-top: 57px;
  margin-top: -57px;
}
.section h3 {
  padding-top: 57px;
  margin-top: -57px;
}
.section h4 {
  padding-top: 57px;
  margin-top: -57px;
}
.section h5 {
  padding-top: 57px;
  margin-top: -57px;
}
.section h6 {
  padding-top: 57px;
  margin-top: -57px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Tsung-Heng Tsai</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="res.html">Research</a>
</li>
<li>
  <a href="pubs.html">Publications</a>
</li>
<li>
  <a href="talk.html">Talks</a>
</li>
<li>
  <a href="teach.html">Teaching</a>
</li>
<li>
  <a href="cv.html">CV</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">PUBLICATIONS</h1>

</div>


<hr />
<div id="journal-papers" class="section level3">
<h3>Journal Papers</h3>
<ul>
<li><p><strong>T-H Tsai</strong>, M Choi, B Banfai, Y Liu, B. MacLean, T Dunkley, and O Vitek (2020). <a href="https://doi.org/10.1074/mcp.RA119.001792">Selection of features with consistent profiles improves relative protein quantification in mass spectrometry experiments</a>. <em>Molecular &amp; Cellular Proteomics</em>, in press.</p></li>
<li><p>BV Sinn, C Fu, R Lau, J Litton, <strong>T-H Tsai</strong>, R Murthy, A Tam, E Andreopoulou, Y Gong, R Murthy, R Gould, Y Zhang, TA King, A Viale, V Andrade, D Giri, R Salgado, I Laios, C Sotiriou, C Marginean, DN Kwiatkowski, R Layman, D Booser, C Hatzis, V Valero, and WF Symmans (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31231679">SET<span class="math inline">\(_{\text{ER/PR}}\)</span> - A robust 18-gene predictor for sensitivity to endocrine therapy for metastatic breast cancer</a>. <em>npj Breast Cancer</em>, 5:16.</p></li>
<li><p><strong>T-H Tsai</strong>, Z Hao, Q Hong, B Moore, C Stella, JH Zhang, Y Chen, M Kim, T Koulis, GA Ryslik, E Verschueren, F Jacobson, WE Haskins, and O Vitek (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28801661">Statistical characterization of therapeutic protein modifications</a>. <em>Scientific Reports</em>, 7(1):7896.</p></li>
<li><p>M Wang, <strong>T-H Tsai</strong>, C Di Poto, A Ferrarini, G Yu, and HW Ressom (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27535232">Topic model-based mass spectrometric data analysis in cancer biomarker discovery studies</a>. <em>BMC Genomics</em>, 17(Suppl 4):545.</p></li>
<li><p><strong>T-H Tsai</strong>, E Song, Z Rui, C Di Poto, M Wang, Y Luo, RS Varghese, MG Tadesse, DH Ziada, CS Desai, K Shetty, Y Mechref, and HW Ressom (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25778709">LC-MS/MS based serum proteomics for identification of candidate biomarkers for hepatocellular carcinoma</a>. <em>Proteomics</em>, 15(13):2369–2381.</p></li>
<li><p><strong>T-H Tsai</strong>, M Wang, C Di Poto, Y Hu, S Zhou, Y Zhao, RS Varghese, Y Luo, MG Tadesse, DH Ziada, CS Desai, K Shetty, Y Mechref, and HW Ressom (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25077556">LC-MS profiling of N-glycans derived from human serum samples for biomarker discovery in hepatocellular carcinoma</a>. <em>Journal of Proteome Research</em>, 13(11):4859–4868.</p></li>
<li><p><strong>T-H Tsai</strong>, MG Tadesse, C Di Poto, LK Pannell, Y Mechref, Y Wang, and HW Ressom (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24013927">Multi-profile Bayesian alignment model for LC-MS data analysis with integration of internal standards</a>. <em>Bioinformatics</em>, 29(21):2774–2780.</p></li>
<li><p><strong>T-H Tsai</strong>, MG Tadesse, Y Wang, and HW Ressom (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23929872">Profile-based LC-MS data alignment — A Bayesian approach</a>. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>, 10(2):494–503.</p></li>
<li><p>JF Xiao, RS Varghese, B Zhou, MR Ranjbar, Y Zhao, <strong>T-H Tsai</strong>, C Di Poto, J Wang, D Goerlitz, Y Luo, AK Cheema, N Sarhan, H Soliman, MG Tadesse, DH Ziada, and HW Ressom (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23078175">LC-MS based serum metabolomics for identification of hepatocellular carcinoma biomarkers in Egyptian cohort</a>. <em>Journal of Proteome Research</em>, 11(12):5914–5923.</p></li>
<li><p>HW Ressom, JF Xiao, L Tuli, RS Varghese, B Zhou, <strong>T-H Tsai</strong>, MR Ranjbar, Y Zhao, J Wang, C Di Poto, AK Cheema, MG Tadesse, R Goldman, and K Shetty (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22882828">Utilization of metabolomics to identify serum biomarkers for hepatocellular carcinoma in patients with liver cirrhosis</a>. <em>Analytica Chimica Acta</em>, 743:90–100.</p></li>
<li><p>L Tuli, <strong>T-H Tsai</strong>, RS Varghese, JF Xiao, AK Cheema, and HW Ressom (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22369182">Using a spike-in experiment to evaluate analysis of LC-MS data</a>. <em>Proteome Science</em>, 10(13).</p></li>
<li><p>GK Befekadu, MG Tadesse, <strong>T-H Tsai</strong>, and HW Ressom (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20837998">Probabilistic mixture regression models for alignment of LC-MS data</a>. <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics</em>, 8(5):1417–1424.</p></li>
<li><p>T-T Liu, Y-H Lin, C-S Hung, T-J Liu, Y Chen, Y-C Huang, <strong>T-H Tsai</strong>, H-H Wang, D-W Wang, J-K Wang, Y-L Wang, and C-H Lin (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19421405">A high speed detection platform based on surface-enhanced Raman scattering for monitoring antibiotic-induced chemical changes in bacteria cell wall</a>. <em>PLoS One</em>, 4(5):e5470.</p></li>
<li><p>Y-H Yi, P-Y Ho, T-W Chen, W-J Lin, V Gukassyan, <strong>T-H Tsai</strong>, D-W Wang, T-S Liu, C-Y Tang, SJ Lo, T-Y Chen, F-J Kao, and C-H Lin (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18996841">Membrane targeting and coupling of NHE1-integrin<span class="math inline">\(\alpha_{\text{IIb}}\beta_{3}\)</span>-NCX1 by lipid rafts following integrin-ligand interactions trigger Ca<span class="math inline">\(^{2+}\)</span> oscillations</a>. <em>Journal of Biological Chemistry</em>, 284(6):3855–3864.</p></li>
</ul>
</div>
<div id="book-chapters" class="section level3">
<h3>Book Chapters</h3>
<ul>
<li><p>Y Fu, <strong>T-H Tsai</strong>, C Mao, SK Mun, HW Ressom, M Wang, Z Zhang, Y Wang (2020). <a href="https://www.sciencedirect.com/science/article/pii/B9780128160343000031">Biological Computing</a>. In <em>Biomedical Information Technology</em> (Ed. D.D. Feng), Academic Press, pp. 81–104.</p></li>
<li><p>Y Zhao, <strong>T-H Tsai</strong>, C Di Poto, LK Pannell, MG Tadesse, HW Ressom (2017). <a href="https://link.springer.com/chapter/10.1007/978-3-319-45809-0_9">Variability Assessment of Label-free LC-MS Experiments for Difference Detection</a>. In <em>Statistical Analysis of Proteomics, Metabolomics and Lipidomics Data using Mass Spectrometry</em> (Eds. S. Datta and B.J.A. Mertens), Springer, pp. 157–176.</p></li>
<li><p><strong>T-H Tsai</strong>, M Wang, and HW Ressom (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26519169">Preprocessing and Analysis of LC-MS-based Proteomic Data</a>. In <em>Statistical Analysis in Proteomics</em> (Ed. K. Jung), Springer, pp. 63–76.</p></li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
