<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>PUBLICATIONS</title>

<script src="site_libs/header-attrs-2.16/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Tsung-Heng Tsai</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="res.html">Research</a>
</li>
<li>
  <a href="pubs.html">Publications</a>
</li>
<li>
  <a href="talk.html">Talks</a>
</li>
<li>
  <a href="teach.html">Teaching</a>
</li>
<li>
  <a href="cv.html">CV</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">PUBLICATIONS</h1>

</div>


<hr />
<div id="journal-papers" class="section level3">
<h3>Journal Papers</h3>
<ul>
<li><p>V Verma, D Dahal, R Radha Krishnan, B Lussem, <strong>T-H
Tsai</strong> (2022). Accurate and reproducible mobility estimation
enables high-throughput analysis of organic field-effect transistors.
<em>Computational Material Science</em>, 214:111689. [<a
href="https://doi.org/10.1016/j.commatsci.2022.111689">Link</a>]</p></li>
<li><p>J Bao, S Jegede, J Hawks, B Dade, Q Guan, S Middaugh, Z Qiu, A
Levina, <strong>T-H Tsai</strong> (2022). Detecting cochlear
synaptopathy through curvature quantification of the auditory brainstem
response. <em>Frontiers in Cellular Neuroscience</em>, 16:851500. [<a
href="https://doi.org/10.3389/fncel.2022.851500">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/35356798/">PubMed</a>]</p></li>
<li><p>T Maculins, E Verschueren, T Hinkle, M Choi, P Chang, C Chalouni,
S Rao, Y Kwon, J Lim, AK Katakam, RC Kunz, BK Erickson, T Huang,
<strong>T-H Tsai</strong>, O Vitek, M Reichelt, Y Senbabaoglu, B
Mckenzie, J Rohde, I Dikic, D Kirkpatrick, A Murthy (2021). Multiplexed
proteomics of autophagy deficient murine macrophages reveals enhanced
antimicrobial immunity via the oxidative stress response.
<em>eLife</em>, 10:e62320. [<a
href="https://elifesciences.org/articles/62320">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/34085925/">PubMed</a>]</p></li>
<li><p>A Arias-Alvarado, M Agayev, S Ilchenko, N Rachdaoui, J Lepp,
<strong>T-H Tsai</strong>, G Zhang, S Previs, T Kasumov (2021).
Measuring acetyl-CoA and acetylated histone turnover in vivo: effect of
a high fat diet. <em>Analytical Biochemistry</em>, 615:114067. [<a
href="https://doi.org/10.1016/j.ab.2020.114067">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/33340539/">PubMed</a>]</p></li>
<li><p>M Choi, J Carver, C Chiva, M Tzouros, T Huang, <strong>T-H
Tsai</strong>, B Pullman, O Bernhardt, R Huttenhain, G Teo, Y
Perez-Riverol, J Muntel, M Muller, S Goetze, M Pavlou, E Verschueren, B
Wollscheid, A Nesvizhskii, L Reiter, T Dunkley, E Sabido, N Bandeira, O
Vitek (2020). MassIVE.quant: a community resource of quantitative mass
spectrometry-based proteomics datasets. <em>Nature Methods</em>,
17:981–984. [<a
href="https://doi.org/10.1038/s41592-020-0955-0">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/32929271/">PubMed</a>]</p></li>
<li><p><strong>T-H Tsai</strong>, M Choi, B Banfai, Y Liu, B. MacLean, T
Dunkley, O Vitek (2020). Selection of features with consistent profiles
improves relative protein quantification in mass spectrometry
experiments. <em>Molecular &amp; Cellular Proteomics</em>,
19(6):944–959. [<a
href="https://doi.org/10.1074/mcp.ra119.001792">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/32234965/">PubMed</a>]</p></li>
<li><p>BV Sinn, C Fu, R Lau, J Litton, <strong>T-H Tsai</strong>, R
Murthy, A Tam, E Andreopoulou, Y Gong, R Murthy, R Gould, Y Zhang, TA
King, A Viale, V Andrade, D Giri, R Salgado, I Laios, C Sotiriou, C
Marginean, DN Kwiatkowski, R Layman, D Booser, C Hatzis, V Valero, WF
Symmans (2019). SET<span class="math inline">\(_{\text{ER/PR}}\)</span>
- A robust 18-gene predictor for sensitivity to endocrine therapy for
metastatic breast cancer. <em>npj Breast Cancer</em>, 5:16. [<a
href="https://doi.org/10.1038/s41523-019-0111-0">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/31231679/">PubMed</a>]</p></li>
<li><p><strong>T-H Tsai</strong>, Z Hao, Q Hong, B Moore, C Stella, JH
Zhang, Y Chen, M Kim, T Koulis, GA Ryslik, E Verschueren, F Jacobson, WE
Haskins, O Vitek (2017). Statistical characterization of therapeutic
protein modifications. <em>Scientific Reports</em>, 7(1):7896. [<a
href="https://doi.org/10.1038/s41598-017-08333-y">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/28801661/">PubMed</a>]</p></li>
<li><p>M Wang, <strong>T-H Tsai</strong>, C Di Poto, A Ferrarini, G Yu,
HW Ressom (2016). Topic model-based mass spectrometric data analysis in
cancer biomarker discovery studies. <em>BMC Genomics</em>, 17(Suppl
4):545. [<a href="https://doi.org/10.1186/s12864-016-2796-x">Link</a>]
[<a
href="https://pubmed.ncbi.nlm.nih.gov/27535232/">PubMed</a>]</p></li>
<li><p><strong>T-H Tsai</strong>, E Song, Z Rui, C Di Poto, M Wang, Y
Luo, RS Varghese, MG Tadesse, DH Ziada, CS Desai, K Shetty, Y Mechref,
HW Ressom (2015). LC-MS/MS based serum proteomics for identification of
candidate biomarkers for hepatocellular carcinoma. <em>Proteomics</em>,
15(13):2369–2381. [<a
href="https://doi.org/10.1002/pmic.201400364">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/25778709/">PubMed</a>]</p></li>
<li><p><strong>T-H Tsai</strong>, M Wang, C Di Poto, Y Hu, S Zhou, Y
Zhao, RS Varghese, Y Luo, MG Tadesse, DH Ziada, CS Desai, K Shetty, Y
Mechref, HW Ressom (2014). LC-MS profiling of N-glycans derived from
human serum samples for biomarker discovery in hepatocellular carcinoma.
<em>Journal of Proteome Research</em>, 13(11):4859–4868. [<a
href="https://doi.org/10.1021/pr500460k">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/25077556/">PubMed</a>]</p></li>
<li><p><strong>T-H Tsai</strong>, MG Tadesse, C Di Poto, LK Pannell, Y
Mechref, Y Wang, HW Ressom (2013). Multi-profile Bayesian alignment
model for LC-MS data analysis with integration of internal standards.
<em>Bioinformatics</em>, 29(21):2774–2780. [<a
href="https://doi.org/10.1093/bioinformatics/btt461">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/24013927/">PubMed</a>]</p></li>
<li><p><strong>T-H Tsai</strong>, MG Tadesse, Y Wang, HW Ressom (2013).
Profile-based LC-MS data alignment — A Bayesian approach. <em>IEEE/ACM
Transactions on Computational Biology and Bioinformatics</em>,
10(2):494–503. [<a href="https://doi.org/10.1109/tcbb.2013.25">Link</a>]
[<a
href="https://pubmed.ncbi.nlm.nih.gov/23929872/">PubMed</a>]</p></li>
<li><p>JF Xiao, RS Varghese, B Zhou, MR Ranjbar, Y Zhao, <strong>T-H
Tsai</strong>, C Di Poto, J Wang, D Goerlitz, Y Luo, AK Cheema, N
Sarhan, H Soliman, MG Tadesse, DH Ziada, HW Ressom (2012). LC-MS based
serum metabolomics for identification of hepatocellular carcinoma
biomarkers in Egyptian cohort. <em>Journal of Proteome Research</em>,
11(12):5914–5923. [<a href="https://doi.org/10.1021/pr300673x">Link</a>]
[<a
href="https://pubmed.ncbi.nlm.nih.gov/23078175/">PubMed</a>]</p></li>
<li><p>HW Ressom, JF Xiao, L Tuli, RS Varghese, B Zhou, <strong>T-H
Tsai</strong>, MR Ranjbar, Y Zhao, J Wang, C Di Poto, AK Cheema, MG
Tadesse, R Goldman, K Shetty (2012). Utilization of metabolomics to
identify serum biomarkers for hepatocellular carcinoma in patients with
liver cirrhosis. <em>Analytica Chimica Acta</em>, 743:90–100. [<a
href="https://doi.org/10.1016/j.aca.2012.07.013">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/22882828/">PubMed</a>]</p></li>
<li><p>L Tuli, <strong>T-H Tsai</strong>, RS Varghese, JF Xiao, AK
Cheema, HW Ressom (2012). Using a spike-in experiment to evaluate
analysis of LC-MS data. <em>Proteome Science</em>, 10(13). [<a
href="https://doi.org/10.1186/1477-5956-10-13">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/22369182/">PubMed</a>]</p></li>
<li><p>GK Befekadu, MG Tadesse, <strong>T-H Tsai</strong>, HW Ressom
(2011). Probabilistic mixture regression models for alignment of LC-MS
data. <em>IEEE/ACM Transactions on Computational Biology and
Bioinformatics</em>, 8(5):1417–1424. [<a
href="https://doi.org/10.1109/tcbb.2010.88">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/20837998/">PubMed</a>]</p></li>
<li><p>T-T Liu, Y-H Lin, C-S Hung, T-J Liu, Y Chen, Y-C Huang,
<strong>T-H Tsai</strong>, H-H Wang, D-W Wang, J-K Wang, Y-L Wang, C-H
Lin (2009). A high speed detection platform based on surface-enhanced
Raman scattering for monitoring antibiotic-induced chemical changes in
bacteria cell wall. <em>PLoS One</em>, 4(5):e5470. [<a
href="https://doi.org/10.1371/journal.pone.0005470">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/19421405/">PubMed</a>]</p></li>
<li><p>Y-H Yi, P-Y Ho, T-W Chen, W-J Lin, V Gukassyan, <strong>T-H
Tsai</strong>, D-W Wang, T-S Liu, C-Y Tang, SJ Lo, T-Y Chen, F-J Kao,
C-H Lin (2009). Membrane targeting and coupling of NHE1-integrin<span
class="math inline">\(\alpha_{\text{IIb}}\beta_{3}\)</span>-NCX1 by
lipid rafts following integrin-ligand interactions trigger Ca<span
class="math inline">\(^{2+}\)</span> oscillations. <em>Journal of
Biological Chemistry</em>, 284(6):3855–3864. [<a
href="https://doi.org/10.1074/jbc.m804334200">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/18996841/">PubMed</a>]</p></li>
</ul>
</div>
<div id="book-chapters" class="section level3">
<h3>Book Chapters</h3>
<ul>
<li><p>Y Fu, <strong>T-H Tsai</strong>, C Mao, SK Mun, HW Ressom, M
Wang, Z Zhang, Y Wang (2020). Biological Computing. In <em>Biomedical
Information Technology</em> (Ed. D.D. Feng), Academic Press, pp. 81–104.
[<a
href="https://www.sciencedirect.com/science/article/pii/B9780128160343000031">Link</a>]</p></li>
<li><p>Y Zhao, <strong>T-H Tsai</strong>, C Di Poto, LK Pannell, MG
Tadesse, HW Ressom (2017). Variability Assessment of Label-free LC-MS
Experiments for Difference Detection. In <em>Statistical Analysis of
Proteomics, Metabolomics and Lipidomics Data using Mass
Spectrometry</em> (Eds. S. Datta and B.J.A. Mertens), Springer,
pp. 157–176. [<a
href="https://link.springer.com/chapter/10.1007/978-3-319-45809-0_9">Link</a>]</p></li>
<li><p><strong>T-H Tsai</strong>, M Wang, HW Ressom (2016).
Preprocessing and Analysis of LC-MS-based Proteomic Data. In
<em>Statistical Analysis in Proteomics</em> (Ed. K. Jung), Springer,
pp. 63–76. [<a
href="https://dx.doi.org/10.1007/978-1-4939-3106-4_3">Link</a>] [<a
href="https://pubmed.ncbi.nlm.nih.gov/26519169/">PubMed</a>]</p></li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
