<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>RESEARCH</title>

<script src="site_libs/header-attrs-2.16/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/readable.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Tsung-Heng Tsai</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="res.html">Research</a>
</li>
<li>
  <a href="pubs.html">Publications</a>
</li>
<li>
  <a href="talk.html">Talks</a>
</li>
<li>
  <a href="teach.html">Teaching</a>
</li>
<li>
  <a href="cv.html">CV</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">RESEARCH</h1>

</div>


<hr />
<div
id="selection-of-informative-features-for-relative-protein-quantification"
class="section level4">
<h4>Selection of informative features for relative protein
quantification</h4>
<div class="row">
<div class="column-left">
<p><img src="images/uninf.png" /></p>
</div>
<div class="column-right">
<p>Summaries of protein abundance are undermined by mass spectrometric
features inconsistent with the overall protein profile. We develop an
automated statistical approach that detects such inconsistent features.
These features can be separately investigated, curated, or removed.
Accounting for such inconsistent features improves the estimation and
the detection of differential protein abundance between conditions. The
approach is implemented as an option in the open-source R-based software
MSstats.</p>
<p><a
href="https://www.ncbi.nlm.nih.gov/pubmed/32234965">Manuscript</a>, <a
href="https://github.com/tsunghengtsai/MSstats-feature-selection">Supplemental</a></p>
</div>
</div>
<hr />
</div>
<div
id="statistical-characterization-of-post-translational-modifications"
class="section level4">
<h4>Statistical characterization of post-translational
modifications</h4>
<div class="row">
<div class="column-left">
<p><img src="images/ptm.png" /></p>
</div>
<div class="column-right">
<p>Interpretation of changes at a single PTM site in a typical bottom-up
proteomics workflow is complicated by sparse PTM coverage and confounded
by both changes in overall protein abundance and variability in
enrichment efficiency. We propose an alternative statistical approach to
model relative abundance changes for modification sites, which
explicitly incorporates major sources of variability and confounding
factors present in PTM experiments. The general statistical framework
underlying the proposed approach also allows for natural extensions to
complex experimental designs including multiple conditions and multiple
batches. All algorithms and statistical tests are implemented in the
MSstatsPTM R package.</p>
<p><a
href="https://www.ncbi.nlm.nih.gov/pubmed/36496144">Manuscript</a></p>
</div>
</div>
<hr />
</div>
<div
id="statistical-characterization-of-therapeutic-protein-modifications"
class="section level4">
<h4>Statistical characterization of therapeutic protein
modifications</h4>
<div class="row">
<div class="column-left">
<p><img src="images/siteocc.png" /></p>
</div>
<div class="column-right">
<p>This study addresses three primary goals of therapeutic protein
characterization: (1) the objective estimation of site occupancyin a
particularcondition, (2) the determination of systematic changes in site
occupancies (i.e., differential site occupancy) between conditions, and
(3) the estimation of combined occupancy over multiple sites. We
demonstrated the importance of appropriate statistical modeling, for
reproducible, accurate and efficient site occupancy estimation and
differential analysis.</p>
<p><a
href="https://www.ncbi.nlm.nih.gov/pubmed/28801661">Manuscript</a></p>
</div>
</div>
<hr />
</div>
<div id="bayesian-alignment-model-for-analysis-of-lc-ms-based-omic-data"
class="section level4">
<h4>Bayesian alignment model for analysis of LC-MS based omic data</h4>
<div class="row">
<div class="column-left">
<p><img src="images/bam.png" /></p>
</div>
<div class="column-right">
<p>Retention time alignment is crucial for the analysis of LC-MS based
omic data. The proposed Bayesian alignment model provides estimates of
the retention time variability along with uncertainty measures, enabling
integration of multiple sources of information including internal
standards and clustered chromatograms. This model has been successfully
applied to data from LC-MS proteomic, metabolomic and glycomic
studies.</p>
<p><a
href="https://www.ncbi.nlm.nih.gov/pubmed/24013927">Manuscript</a>, <a
href="https://github.com/tsunghengtsai/alignLCMS">Supplemental</a></p>
</div>
</div>
<hr />
</div>
<div id="profile-based-lc-ms-data-alignment" class="section level4">
<h4>Profile-based LC-MS data alignment</h4>
<div class="row">
<div class="column-left">
<p><img src="images/dag.png" /></p>
</div>
<div class="column-right">
<p>This study provides methodological foundation for the Bayesian
alignment model and demonstrates a proof of concept for profile-based
data alignment. The proposed model uses Markov chain Monte Carlo (MCMC)
methods to draw inference on model parameters and improves on existing
MCMC based alignment methods through 1) the implementation of an
efficient MCMC sampler and 2) an adaptive selection of knots using
stochastic search variable selection (SSVS).</p>
<p><a
href="https://www.ncbi.nlm.nih.gov/pubmed/23929872">Manuscript</a></p>
</div>
</div>
<hr />
</div>
<div id="lc-ms-based-serum-glycomics-for-hcc-biomarker-discovery"
class="section level4">
<h4>LC-MS based serum glycomics for HCC biomarker discovery</h4>
<div class="row">
<div class="column-left">
<p><img src="images/glycan.png" /></p>
</div>
<div class="column-right">
<p>Hepatocellular carcinoma (HCC) is the third leading cause of cancer
mortality worldwide with five-year survival rate less than 15%.
Glycosylation has important implications in the development and
progression of cancer. In collaboration with analytical chemists at TTU,
we used two complementary LC-MS methods, global profiling and targeted
quantitation, to analyze serum N-glycans from hundreds of participants
recruited in Egypt and the US, and identified candidate biomarkers that
distinguishing HCC cases from patients with liver cirrhosis. In this
study, I led the effort on quantitative analysis including development
of preprocessing pipelines and statistical analysis.</p>
<p><a
href="https://www.ncbi.nlm.nih.gov/pubmed/25077556">Manuscript</a></p>
</div>
</div>
<hr />
</div>
<div id="lc-ms-based-serum-proteomics-for-hcc-biomarker-discovery"
class="section level4">
<h4>LC-MS based serum proteomics for HCC biomarker discovery</h4>
<div class="row">
<div class="column-left">
<p><img src="images/pmic.png" /></p>
</div>
<div class="column-right">
<p>In this proteomic study, I led the effort on study design and data
analysis including initial assessment of different analytical protocols
and platforms. We identified candidate protein biomarkers through
LC-MS/MS based untargeted analysis of sera. These proteins were further
evaluated by targeted quantitation using MRM. Our findings independently
confirmed several previously reported HCC protein biomarkers and
revealed a number of new candidate biomarkers. Among these, there were
21 candidate protein biomarkers found significant in our two study
cohorts in Egypt and the US. Furthermore, through multivariate analysis
we identified a panel of six protein biomarkers, which yielded better
performance than alpha-fetoprotein (AFP), the HCC biomarker in current
use.</p>
<p><a
href="https://www.ncbi.nlm.nih.gov/pubmed/25778709">Manuscript</a></p>
</div>
</div>
<hr />
</div>
<div id="lc-ms-based-serum-metabolomics-for-hcc-biomarker-discovery"
class="section level4">
<h4>LC-MS based serum metabolomics for HCC biomarker discovery</h4>
<div class="row">
<div class="column-left">
<p><img src="images/metabo.png" /></p>
</div>
<div class="column-right">
<p>I contributed to the analysis of LC-MS metabolomic data for HCC
biomarker discovery through development of outlier screening method
using the Stahel-Donoho robust estimator and application of XCMS
preprocessing pipelines. As in our glycomic and proteomic studies, the
metabolomic analysis was performed on two study cohorts in Egypt and the
US. The three types of omic measurements conducted on the same subjects
present an interesting yet challenging opportunity to integrate
multi-omic signatures for a more comprehensive characterization of
HCC.</p>
<p><a href="https://www.ncbi.nlm.nih.gov/pubmed/22882828">Manuscript
1</a>, <a href="https://www.ncbi.nlm.nih.gov/pubmed/23078175">Manuscript
2</a></p>
</div>
</div>
<hr />
</div>
<div id="spike-in-experiment-for-evaluation-of-preprocessing-pipelines"
class="section level4">
<h4>Spike-in experiment for evaluation of preprocessing pipelines</h4>
<div class="row">
<div class="column-left">
<p><img src="images/spike.png" /></p>
</div>
<div class="column-right">
<p>Prior to our HCC biomarker discovery studies, we conducted critical
assessment on existing preprocessing pipelines for LC-MS data analysis.
We simulated a simple model with “presence” and “absence” condition
using spike-in experiments, and evaluated tools on the basis of their
capabilities to identify the true differences. This study highlights
critical preprocessing steps and helps prioritize improvement on
relevant analytical and computational approaches, which has greatly
facilitated our subsequent omic studies for biomarker discovery.</p>
<p><a
href="https://www.ncbi.nlm.nih.gov/pubmed/22369182">Manuscript</a></p>
</div>
</div>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
